### PR TITLE
AC Automated Test LRAC-8412 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**
https://issues.liferay.com/browse/LRAC-8412
https://issues.liferay.com/browse/LRAC-8765 (refactor)

### CODE REVIEW

**Comment by LucasAcacio** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase [hunk 1]
```diff
@@ -32,16 +32,17 @@ definition {
...
-			JSONGroup.deleteGroupByName(groupName = "Site Name");
-		}
+		//if ("${testPortalInstance}" == "true") {
+		//	PortalInstances.tearDownCP();
+		//}
+		//else {
+		//	JSONGroup.deleteGroupByName(groupName = "Site Name");
+		//}
+
+		//ACUtils.tearDownAC();
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:44]: hi @LucasAcacio , please remove this comment.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase [hunk 2]
```diff
@@ -282,12 +283,55 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8412 | Test Summary: Documents location card shows downloads by the expected location"
-	@ignore = "true"
 	@priority = "5"
 	test DocumentsLocationCardShowsViewsByExpectedLocation {
+		property portal.upstream = "quarantine";
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:288]: hi @LucasAcacio , could you please try to update this test case using this logic:

https://github.com/liferay-qa/liferay-portal/pull/689/commits/b189cd8ff1a93c59b879ffe22f05e44c143c91c5

https://github.com/liferay-qa/liferay-portal/pull/689/commits/224057c4bd3b8aea52e90146eb481acac1edbf64

Thanks!

**Comment by alexcandidos** [comment]
In this case, we don't need to add this on quarantine.

**Pull request closed without merging**